src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
```

Regarding padding. [The requirement](https://github.com/mate-academy/layout_catalog-test_task#requirements) says this should be applied to top and bottom, not left and right + there is already a `margin: auto` applied for centring the container
```suggestion
  padding: 120px 0;
```